README.md

```diff
@@ -4,6 +4,7 @@ The default template project for [nano-react-app](https://github.com/nano-react-
 
 - `npm start` — This will spawn a development server with a default port of `3000`.
 - `npm run build` — This will output a production build in the `dist` directory.
+- `npm run preview` — This will run the production build locally with a default port of `5173` (this will not work if you haven't generated the production build yet)
```

Comment:
Add period for consistency

```suggestion
- `npm run preview` — This will run the production build locally with a default port of `5173` (this will not work if you haven't generated the production build yet).
```